src/openms/source/APPLICATIONS/ConsoleUtils.cpp

```diff
@@ -121,6 +120,7 @@ namespace OpenMS
         {
           LOG_DEBUG << "output shaping: stty size command failed." << std::endl;
         }
+        pclose(fp);
```

Comment:
> well, introducing a segfault to "fix" a coverity error seems not really a gain
this is why I suggested a revert. 
We wont get the coverity issue fixed like that anyways.